routes/index.js

```diff
@@ -28,7 +32,10 @@ module.exports = function(utils, env, nunjucksEnv, appName) {
       previewLoader = env.get("PREVIEW_LOADER"),
       together = env.get("USE_TOGETHERJS") ? env.get("TOGETHERJS") : false,
       userbarEndpoint = env.get("USERBAR"),
-      webmaker = env.get("WEBMAKER_URL");
+      webmaker = env.get("WEBMAKER_URL"),
+      oauthClientId = env.get("OAUTH_CLIENT_ID"),
```

Comment:
using habitat's prefix object fetching (`env.get("oauth")` => `oauth` object with `.client_id`, `.client_secret`, etc) might be better